src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -21,6 +21,8 @@
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableMap;
 
+import org.jabref.logic.importer.fetcher.CrossRef;
+import org.jabref.logic.importer.fileformat.bibtexml.Inproceedings;
```

Comment:
Please check your imports, it seems like you have imported the type from the wrong package.
And CrossRef here looks also odd.